YACS-RCOS/yacs.n
# Color Mode Button/Notification UI Changed

- Changed the notification button and moon to a dropdown list so that it's more intuitive to the user
- Added brief text in front so the user instantly knows the function of the list
- Moved toast notification to the top-center of the screen so that it doesn't interfere with switching between themes

**Issue**
> closes #651

**Database Changes/Migrations**

Nothing changed here 

**Test Modifications**

Nothing changed here 

**Test Procedure**
- Use the new switch to change from light/dark mode
- Double click moon to see the notification moved to the top-middle of the screen, allowing you to adjust your settings without notification in your way

**Photos**
Old
![image](https://user-images.githubusercontent.com/91478985/221088098-2e42659a-842c-47a7-9a19-10f7f619f955.png)

New
![image](https://user-images.githubusercontent.com/91478985/227634522-af7837a8-794c-4ff3-a2c9-db3bc9fdaa3c.png)
![image](https://user-images.githubusercontent.com/91478985/227634568-61f169fb-e8c9-4ef4-be7e-cbce95c6c503.png)

Old
![image](https://user-images.githubusercontent.com/91478985/221088816-f1e4f278-1e0b-4a6c-9911-41ab2e43452d.png)

New 
![image](https://user-images.githubusercontent.com/91478985/224866058-c2b71545-7e0e-4fa9-91a4-1e0a4b9f80cc.png)

**Additional Info**
